### PR TITLE
Add Brexit source

### PR DESCRIPTION
**Pull request by sumeetadur**
To be used in the upcoming Brexit front page component

### CODE REVIEW

**Comment by sumeetadur** [comment]
@ironsidevsquincy could you have a look through this please?

**Comment by matthew-andrews** [comment]
darren is on holiday at the moment, perhaps @aintgoin2goa or @andygout can help?

File: server/config/sources.js [hunk 1]
```diff
@@ -48,6 +48,10 @@ const sources = {
 	editorsPicks: {
 		type: 'list',
 		uuid: '73667f46-1a55-11e5-a130-2e7db721f996'
+	},
+	brexit: {
+		type: 'list',
+		uuid: '7a084d44-b95a-11e5-b151-8e15c9a029fb' //TODO: Use correct methode list
```
Comment (andygout) [inline server/config/sources.js]: Where does this `uuid` currently point to? I notice it's not used by any other sources in the file but is still marked as a 'TO DO' in terms of correcting it. Just curious.

File: server/lib/schema.js [hunk 2]
```diff
@@ -17,6 +17,13 @@ const queryType = new GraphQLObjectType({
 	name: 'Query',
 	description: 'FT content API',
 	fields: {
+		brexit: {
+			type: List,
+			resolve: (root, _, { rootValue: { flags, backend = backendReal }}) => {
+				const uuid = sources['brexit'].uuid;
+				return backend(flags).capi.list(uuid);
+			}
+		},
```
Comment (andygout) [inline server/lib/schema.js]: Just a heads-up that you'll likely have noticed that as in Opinion (https://github.com/Financial-Times/next-graphql-api/blob/master/server/lib/schema.js#L64-L70) you can add a URL as an argument here if the frontpage Brexit component links to a Brexit section (realise I hadn't asked that).

**Comment by andygout** [comment]
👍

File: server/config/sources.js (hunk 1 above)
Comment (sumeetadur) [inline server/config/sources.js]: It's pointing to the methode list currently used on the Brexit stream page in the stickies component:
screen shot 2016-05-18 at 12 43 33

Comment (andygout) [inline server/config/sources.js]: Cool.

**Pull request merged**